dissect/target/plugins/apps/browser/firefox.py

```diff
@@ -74,6 +78,47 @@ def _iter_db(self, filename: str) -> Iterator[SQLite3]:
                     except SQLError as e:
                         self.target.log.warning("Could not open %s file: %s", filename, db_file, exc_info=e)
 
+    @export(record=BrowserCookieRecord)
+    def cookies(self) -> Iterator[BrowserCookieRecord]:
+        """Return browser cookies records from Firefox.
+        Args:
+            browser_name: The name of the browser as a string.
+        Yields:
+            Records with the following fields:
+                ts_created (datetime): Download start timestamp.
+                ts_last_accessed (datetime): Download end timestamp.
```

Comment:
Same here. Is this the correct description? Should this not be related to the creation and last access of the cookie?